tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Auth.java

```diff
@@ -137,6 +136,20 @@ public Response login(
     }
 
 
+    /**
+     * Create a LoginResponse, given a session.
+     * Put this here and not in LoginResponse because of serialization
+     * @param session
+     * @return
+     */
+    private LoginResponse createLoginResponse(CookieSession session) {
+        LoginResponse resp = new LoginResponse();
+        resp.sessionId = session.id;
+        resp.user = (session.user != null);
+        return resp;
```

Comment:
it could, but what I'd like to do is (re)add user as a full structure.  So `if (resp.user)` is future proof on the FE, as it is either present or absent.